src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -536,7 +532,7 @@ kbd {
 }
 
 #theme-picker, #settings-menu, #help-button {
-	border-color: #5c6773;
+	border-color: #424c57;
```

Comment:
Maybe group this one with `.search-input`? Like that, it won't be an issue ever again.